doc/SONiC_YANG_Model_Guidelines.md

```diff
@@ -268,3 +649,372 @@ container PORTCHANNEL_INTERFACE {
         }
 }
 ```
+
+### 22. In some cases it may be required to split an ABNF table into multiple YANG lists based on the data stored in the ABNF table. Use "sonic-ext:table" extension if list name is not same as ABNF table name.
```

Comment:
Do we really need sonic-ext:table annotation here? Table name is already provided by the parent container.

Reply:
I agree, we can derive from container name also.